app/Commands/Repl.hs

```diff
@@ -60,6 +60,7 @@ helpTxt =
   :multiline                Start a multi-line input. Submit with <Ctrl-D>
   :root                     Print the current project root
   :version                  Display the Juvix version
+  :dev        DEV CMD       Command reserved for debugging
```

Comment:
As discussed consider moving this to dev repl

Reply:
this line will only be shown in the dev repl now.